controllers/v1/solutionsController.js

```diff
@@ -1558,5 +1548,54 @@ module.exports = class Solutions extends Abstract {
 
         })
     }
+
+    /**
+    * @api {post} /assessment/api/v1/solutions/addEntities/:solutionId Add entity to solution
+    * @apiVersion 1.0.0
+    * @apiName Add entity to solution
+    * @apiGroup Solutions
+    * @apiParamExample {json} Request-Body:
+    * {
+    *	    "entities": ["5beaa888af0065f0e0a10515","5beaa888af0065f0e0a10516"]
+    * }
+    * @apiHeader {String} X-authenticated-user-token Authenticity token
+    * @apiSampleRequest /assessment/api/v1/solutions/addEntities/5f64601df5f6e432fe0f0575
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    * {
+    *   "message" : "Entities updated successfully."
+    * }
+    */
+
+     /**
+   * Add entity to solution.
+   * @method
+   * @name addEntities
+   * @param {Object} req - requested data.
+   * @param {String} req.params._id - solution id.
+   * @returns {JSON} consists message of successfully mapped entities
+   */
+
+  async addEntities(req) {
```

Comment:
@aman-tunerlabs - Pleae add this and the above API to be accessible only if the internal access token is passed.